website/config.toml

```diff
@@ -0,0 +1,3 @@
+baseURL = "http://example.org/"
+languageCode = "en-us"
+title = "My New Hugo Site"
```

Comment:
Let's change the `title` to "Apprenticeships" and `baseURL` to `https://apprenticeships.fvcproductions.com/`